.circleci/src/workflows/commit.yml

```
@@ -1,2 +1,6 @@
 jobs:
   - check
+  - build_android
+  - deliver_android:
+      requires:
+        - build_android
```

```suggestion
        - check
        - build_android
```
Or even make it required for `build_android`

---

good idea. Changed it.